app/models/discourse_activity_pub_activity.rb

```diff
@@ -7,6 +7,22 @@ class DiscourseActivityPubActivity < ActiveRecord::Base
 
   after_create :deliver_composition, if: Proc.new { ap&.composition? }
 
+  attr_accessor :to
+
+  DEFAULT_VISIBILITY = 'private'
```

Comment:
hm, I have a slight disagreement here. The default in the client is public as you say, however the reason I set this constant to private is because a server default should arguably be more conservative. If some exception were to occur you want the least surprise possible. Note that there already is a migration with a default of private (1 is private, 2 is public): https://github.com/discourse/discourse-activity-pub/pull/19/files#diff-c21282442bc42468d5fbc16dbde78a8a554e930d3b9a060640c4dae761979539R2

Reply:
I think what may have been confusing here is that that constant (DEFAULT_VISIBILITY) is really just there to make it easier to apply the visibility column default in other contexts, specifically as the value used in the equivalent custom fields is a string instead of an integer (for various reasons). I've updated the usage here to remove the need for the constant making it clearer that we're dealing with a database default here, i.e. the one you see in the migration: https://github.com/discourse/discourse-activity-pub/pull/19/commits/1068cb402aee9a5e734aca5cd88ce6ac33d8cc51 . We can change this database default to public too if you prefer.

Reply:
Yes, I am a bit confused by the server-side vs client-side inconsistency. 

So, my understanding is this: 

- because of the database default (private) and the migration, all existing AP activities will be marked as private
- in the admin UI, the default is Public
  - for categories not yet set up with AP, this means that if the user makes no changes, the posts will default to public (because the category dropdown is by default set to Public)
  - but for existing AP-configured categories, what happens? UI in the category settings says Public (that's the default), but, without making edits to the category settings, I just tested, and a new topic in that category will have `1` in the visibility column, i.e. it's followers only. There's a disconnect here. 

So, for the existing AP-configured categories, we need something. Either we keep them as "Followers Only" and add a migration for the category setting, to make sure the admin UI says "Followers Only". Or we switch to default public for all existing categories, but need to make sure new posts in these categories are marked as Public. Could do this with a migration or with swapping the server-side default. 

I'm fine with either, slight preference would be for switching to Public, (since it's early days in the life of the plugin, I am ok with the breaking change) via a migration so that DB structure is correct, i.e. all AP-enabled categories would have an explicit entry for the visibility setting. 

Does this make sense @angusmcleod?

Reply:
I can see arguments either way, but I don't feel particularly strongly about this, so let's just make the database default public. Done: https://github.com/discourse/discourse-activity-pub/pull/19/commits/1068cb402aee9a5e734aca5cd88ce6ac33d8cc51

Reply:
I've made the database default public and added a data migration 👍 

https://github.com/discourse/discourse-activity-pub/pull/19/commits/6d6365869c5ebe06b86446213ff242684d41d896

Partly for ease of review, toggling the migration up and down will work:

```
LOAD_PLUGINS=1 rake db:migrate:up VERSION=20230713131335
LOAD_PLUGINS=1 rake db:migrate:down VERSION=20230713131335
```